styhead/rz-sbc: rebase onto latest Dunfell branch

tiennguyenuh opened the pull request:
Rebased the styhead/rz-sbc branch onto the latest dunfell/rz-sbc branch to align with
new changes.

tiennguyenuh committed:
rz-sbc: update readme files

Update readme to clarify the purpose of each folder.

Signed-off-by: Tien Nguyen <tien.nguyen.uh@renesas.com>

tiennguyenuh committed:
rz-sbc: add docker support option

Changes include:
	- Add docker support option in `local.conf.sample`
	- Update docker.cfg

Signed-off-by: Tien Nguyen <tien.nguyen.uh@renesas.com>

tiennguyenuh committed:
rz-sbc: Update renesas-ubuntu.bb

Changes include:
	- Clean duplicate configuration in `renesas-ubuntu.bb` and fix
	  the issue of creating esdk that needs `environment-setup-append.sh`.
	- Add clean up call in `SDK_POSTPROCESS_COMMAND` after building SDK.
	- Move the renesas-ubuntu recipe into each qt version folder.

Signed-off-by: Tien Nguyen <tien.nguyen.uh@renesas.com>

tiennguyenuh committed:
gstreamer-omx: change submodule url

Using a patch to update the submodule URL for gst-omx from
https://gitlab.freedesktop.org/gstreamer/common.git to
https://github.com/GStreamer/common.git

Signed-off-by: Tien Nguyen <tien.nguyen.uh@renesas.com>

tiennguyenuh committed:
rz-sbc: correct addtask ordering in bsp recipes

Correct addtask ordering in firmware-package recipes

Signed-off-by: Tien Nguyen <tien.nguyen.uh@renesas.com>

tiennguyenuh committed:
rz-sbc: rename rzpi to rzg2l in linux recipe

Add patch to rename rzpi to rzg2l-sbc in Linux recipe

Signed-off-by: Tien Nguyen <tien.nguyen.uh@renesas.com>

tiennguyenuh committed:
rz-sbc: python3-pandas: Add "inherit pkgconfig"

Add inherit pkgconfig to fix this issue.

| Did not find pkg-config by name 'pkg-config'
| Found pkg-config: NO
| Run-time dependency python found: NO (tried pkgconfig, pkgconfig and sysconfig)

Signed-off-by: Tien Nguyen <tien.nguyen.uh@renesas.com>

vudangRVC commented on recipes-docs/rzg2l-sbc-readme/files/README.md:
```diff
@@ -48,38 +49,40 @@ After preparing the host machine for building, download necessary packages (get
 - Graphic: https://www.renesas.com/us/en/document/swo/rz-mpu-graphics-library-evaluation-version-rzg2l-and-rzg2lc-rtk0ef0045z13001zj-v112enzip
```
We don't use Graphic libraries anymore, please remove it

tiennguyenuh replied:
Updated in the commit https://github.com/Renesas-SST/meta-renesas/commit/b5c4b9edd93183391913a2aba5ead7779dea9224

vudangRVC commented on recipes-docs/rzg2l-sbc-readme/files/README.md:
````diff
@@ -1330,21 +1468,6 @@ To securely transfer files between local and remote systems, SCP can be used on
       ```
    - Type `yes` to confirm the host's authenticity when prompted.
 
-#### Switching from OpenSSH to Dropbear
````
Dropbear is specifically designed for embedded systems, please do not drop it.
https://layers.openembedded.org/layerindex/recipe/4674/

tiennguyenuh replied:
Updated in the commit https://github.com/Renesas-SST/meta-renesas/commit/b5c4b9edd93183391913a2aba5ead7779dea9224

vudangRVC commented on recipes-docs/rzg2l-sbc-readme/files/README.md:
```diff
@@ -864,10 +932,10 @@ The distribution comes with Debian package manager `apt-get` and `dpkg` for bina
-deb [arch=arm64] http://ports.ubuntu.com/ focal-security main multiverse universe
-deb [arch=arm64] http://ports.ubuntu.com/ focal-backports main multiverse universe
-deb [arch=arm64] http://ports.ubuntu.com/ focal-updates main multiverse universe
+deb [arch=arm64] http://ports.ubuntu.com/ oracular main multiverse universe
```
As dicussed, use the Ubuntu repository if the Debian repository is not supported.

tiennguyenuh replied:
Yes, it's Ubuntu repository. We're still using them by default.

vudangRVC commented on recipes-extend/target-readme/files/images/Readme.md:
```diff
@@ -7,35 +7,76 @@ This directory contains various image files used for deploying and booting the R
+├── rootfs
+│   ├── core-image-bsp-rzg2l-sbc.tar.bz2
+│   ├── core-image-minimal-rzg2l-sbc.tar.bz2
+│   ├── core-image-qt-rzg2l-sbc.tar.bz2
```
This is an old hierarchy

tiennguyenuh replied:
Updated in the commit https://github.com/Renesas-SST/meta-renesas/commit/b5c4b9edd93183391913a2aba5ead7779dea9224

vudangRVC commented on recipes-extend/target-readme/files/images/rootfs/Readme.md:
```diff
@@ -7,12 +7,26 @@ This directory contains the compressed root filesystem image used for deploying
+├── renesas-quickboot-cli-rzg2l-sbc.tar.bz2
+├── renesas-quickboot-wayland-rzg2l-sbc.tar.bz2
+├── renesas-ubuntu-rzg2l-sbc.tar.bz2
+├── ubuntu-core-image-qt-rzg2l-sbc.tar.bz2
```
Rename the ubuntu-core and the ubuntu-lxde image.

tiennguyenuh replied:
Updated in the commit https://github.com/Renesas-SST/meta-renesas/commit/b5c4b9edd93183391913a2aba5ead7779dea9224

tiennguyenuh committed:
rz-sbc: add guide to switch SSH server and add GPG key

Changes include:
	- Add guide to switch from OpenSSH to Dropbear.
	- Install `trusted.gpg` key by default for APT repository.

Signed-off-by: Tien Nguyen <tien.nguyen.uh@renesas.com>

vudangRVC commented on recipes-multimedia/gstreamer/gstreamer1.0-omx_%.bbappend:
This is an empty file. Is it what you want to merge?

tiennguyenuh replied:
It was force-pushed by mistake. I've updated the recipe in the commit: https://github.com/Renesas-SST/meta-renesas/commit/d9d44b67f6c17bfefc6b5f930cb08eded3c54337
